doc/source/ray-overview/installation.rst

```diff
@@ -49,16 +49,17 @@ You can install the nightly Ray wheels via the following links. These daily rele
   # pip install -U LINK_TO_WHEEL.whl
 
 
-====================  ====================  =======================
-       Linux                 MacOS          Windows (beta)
-====================  ====================  =======================
-`Linux Python 3.11`_  `MacOS Python 3.10`_  `Windows Python 3.10`_
-`Linux Python 3.10`_   `MacOS Python 3.9`_   `Windows Python 3.9`_
-`Linux Python 3.9`_   `MacOS Python 3.8`_   `Windows Python 3.8`_
-`Linux Python 3.8`_   `MacOS Python 3.7`_   `Windows Python 3.7`_
-`Linux Python 3.7`_   `MacOS Python 3.6`_
-`Linux Python 3.6`_
-====================  ====================  =======================
+====================                ====================  =======================
+       Linux                        MacOS                 Windows (beta)
+====================                ====================  =======================
+`Linux Python 3.10`_                `MacOS Python 3.10`_  `Windows Python 3.10`_
+`Linux Python 3.9`_                 `MacOS Python 3.9`_   `Windows Python 3.9`_
+`Linux Python 3.8`_                 `MacOS Python 3.8`_   `Windows Python 3.8`_
+`Linux Python 3.7`_                 `MacOS Python 3.7`_   `Windows Python 3.7`_
+`Linux Python 3.6`_                 `MacOS Python 3.6`_
+`Linux Python 3.5`_
+`Linux Python 3.11 (EXPERIMENTAL)`_ 
```

Comment:
I don't think this worked, let me fix

Reply:
Also, Python 3.5 is not supported any more, let's remove that :)

Reply:
fixed, and I also removed 3.5